dev/ThemeStyle.scss

```diff
@@ -0,0 +1,18 @@
+.configurableInteractiveLayout-card {
+    border: solid 2px #000;
+}
+
+.configurableInteractiveLayout-title {
+    color: blue;
+}
+
+.configurableInteractiveLayout-cardsLayoutContainer {
+    background-color: #f3f6f6;
+}
+  
+/* Transparent Card
```

Comment:
if commenting out, do we need to keep it? Why not having it?

Reply:
I had to rewrite the entire style sheet because CSS-Modules wasn't handling the '&' operator properly. I commented it out initially so I wouldn't lose any styling while I rewrote everything. I guess I can delete it all now (if we need anything back, we can always look at history)